app/javascript/packages/document-capture/components/in-person-prepare-step.tsx

```diff
@@ -29,7 +31,7 @@ function InPersonPrepareStep({ toPreviousStep, value }) {
       setIsSubmitting(true);
       removeUnloadProtection();
       await trackEvent('IdV: prepare submitted');
-      window.location.href = (event.target as HTMLAnchorElement).href;
+      setStepName((event.target as HTMLAnchorElement).href);
```

Comment:
I _think_ this change might be uneccessary since the `href` value is set in the `SpinnerButton`, but I'm not totally certain.